### PR TITLE
routing and dispatch routes

### PR DESCRIPTION
**Pull request by FH-sandeepkaur1**

### CODE REVIEW

**Review by amritfleethawks — COMMENTED** [review]
there are conflicts in files, need to discuss with sandeep to merge

File: src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts [hunk 1]
```diff
@@ -36,53 +36,63 @@ export class AddRouteComponent implements OnInit {
...
+  form;
+
+  // routeNo = '';
+  // routeName = '';
+  // notes = '';
+  // VehicleID = '';
+  // AssetID = '';
+  // miles = '';
+  // driverUserName = '';
+  // coDriverUserName = '';
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts:61]: Please remove commented code.

File: src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts [hunk 2]
```diff
@@ -315,6 +327,7 @@ export class AddRouteComponent implements OnInit {
   }
 
   throwErrors() {
+    console.log(this.errors);
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts:330]: remove console.logs

File: src/app/pages/dispatch/trips/add-trip/add-trip.component.ts [hunk 3]
```diff
@@ -0,0 +1,868 @@
...
+
+        // $(document).on('change', '#informationAsset', function () {
+        //     let selected = $("#informationAsset option:selected");
+        //     if(selected.val() !== ''){
+        //         let objj = {
+        //             id: selected.val(),
+        //             name: selected.text()
+        //         }
+        //         console.log('objj');
+        //         console.log(objj);
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/trips/add-trip/add-trip.component.ts:203]: remove commented code if not required

File: src/app/pages/dispatch/trips/add-trip/add-trip.component.ts [hunk 4]
```diff
@@ -0,0 +1,868 @@
...
+            zipcode: '',
+            locationName: '',
+            type: '',
+            index: ''
+        };
+    }
+
+    createTrip() {
+        this.spinner.show();
+        console.log('start tripData');
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/trips/add-trip/add-trip.component.ts:756]: remove console.logs

File: src/app/pages/dispatch/trips/trip-detail/trip-detail.component.ts [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+
+  // initSpeedChart(){
+  //  var graph2 = new Chart('speedChart', {
+  //     type: 'line',
+  //     data: {
+  //        labels: [
+  //            '12:00 AM', '12:30 AM','1:00 AM', '1:30 AM','2:00 AM', '2:30 AM','3:00 AM', '3:30 AM',
+  //            '4:00 AM', '4:30 AM','5:00 AM', '5:30 AM','6:00 AM', '6:30 AM','7:00 AM', '7:30 AM',
+  //            '8:00 AM', '8:30 AM','9:00 AM', '9:30 AM','10:00 AM', '10:30 AM','11:00 AM', '11:30 AM',
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/trips/trip-detail/trip-detail.component.ts:207]: remove commented code if not required

**Review by kunalfleethawks — APPROVED** [review]
Hi @FH-sandeepkaur1 I see lots of code commented please remove it if not required.

**Pull request closed without merging**